jobs/service-fabrik-deployment-hooks/spec

```diff
@@ -0,0 +1,72 @@
+---
+name: service-fabrik-deployment-hooks
+
+packages:
+  - bosh-helpers
+  - node
+  - service-fabrik-deployment-hooks
+  - jq
+  - yaml2json
+
+templates:
+  bin/service-fabrik-deployment-hooks_ctl.erb: bin/service-fabrik-deployment-hooks_ctl
+  bin/job_properties.sh.erb: bin/job_properties.sh
+  config/settings.yml.erb: config/settings.yml
+  config/eventlog-config-hook.yml.erb: config/eventlog-config-hook.yml
+
+provides:
+- name: deployment_hooks
+  type: deployment_hooks
+  properties:
+  - username
+  - password
+  - skip_ssl_validation
```

Comment:
dont provide what you feel will not be used by other jobs. for ex, all these log_level, enable_circuit_breaker etc.. are internal to this job, why bother providing to other jobs which is not going to be used. So provide only the only the ones thats required.

Reply:
Okay. Was not aware of this feature. I though all the properties needs to be in provided array. Will make the correction

Reply:
Done